<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(214, 43, 52, 255), stop:1 rgba(251, 175, 178, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>71</y>
      <width>600</width>
      <height>338</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 255, 255);
border-radius: 20px</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>31</y>
       <width>161</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #BA5D5D</string>
     </property>
     <property name="text">
      <string>Please rate</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblGroupName">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>60</y>
       <width>601</width>
       <height>82</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>70</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #E65353</string>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>REVERSI</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn1">
     <property name="geometry">
      <rect>
       <x>53</x>
       <y>151</y>
       <width>86</width>
       <height>86</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/assets/star-normal.png</normaloff>:/images/assets/star-normal.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>86</width>
       <height>86</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn2">
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>151</y>
       <width>86</width>
       <height>86</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/assets/star-normal.png</normaloff>:/images/assets/star-normal.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>86</width>
       <height>86</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn3">
     <property name="geometry">
      <rect>
       <x>257</x>
       <y>151</y>
       <width>86</width>
       <height>86</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/assets/star-normal.png</normaloff>:/images/assets/star-normal.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>86</width>
       <height>86</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn4">
     <property name="geometry">
      <rect>
       <x>359</x>
       <y>151</y>
       <width>86</width>
       <height>86</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/assets/star-normal.png</normaloff>:/images/assets/star-normal.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>86</width>
       <height>86</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn5">
     <property name="geometry">
      <rect>
       <x>461</x>
       <y>151</y>
       <width>86</width>
       <height>86</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/assets/star-normal.png</normaloff>:/images/assets/star-normal.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>86</width>
       <height>86</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConfirm">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>282</y>
       <width>129</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/assets/button-bg.png);
border-radius: 5px;
color: white;</string>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
